docs/reference/commandline/service_update.md

```diff
@@ -17,6 +17,8 @@ Update a service
 
 Options:
       --args string                    Service command args
+      --cap-add                        Add Linux capabilities (default [])
+      --cap-drop                       Drop Linux capabilities (default [])
```

Comment:
This probably won't work; the `service update` command allows you to specify the _diff_ / changes, compared to the current service specification, so if we use the current convention; this would become `--cap-add-add`, `--cap-add-rm`, and `--cap-drop-add`, `--cap-drop-rm`, which would be really confusing.

We need to come up with different naming for this.

Reply:
For me as user it's OK to have `--cap-add-add`, because it express exact the intention to add or remove something to or from `--cap-add`. Another name I can think of is like `--cap-extra-add`/`--cap-extra-rm`. Or `--cap-include-add`/`--cap-include-rm` and `--cap-exclude-add`/`--cap-exclude-rm`.